doc_admin/AdminManual.adoc

```diff
@@ -381,10 +385,14 @@ NOTE: Folder structure and known issues are found at:
 https://github.com/informatici/openhospital/blob/master/POH-README.md
 
 [#backup-restore]
-=== 3.3 Backup & Restore (alternative with MySQL Workbench®)
+=== 3.3 Backup & Restore
+
+Backup of the Open Hospital database can be performed in several ways. The easiest way is to use the provided scripts (oh.sh/oh.ps1) by choosing the option "-s":
+
+Restore of the database is also possible; See script help for more informations.
```

Comment:
I would reword this to be: `Restoring the database is also possible; see the script help for more information.`

Side note: I find it interesting the number of times I run into non-native English speakers making `information` plural (by adding the `s`); `information` in English is both singular and plural.   One of the many things that makes English so hard to learn.

Reply:
And the funny thing is that I know this, I think about it, and then I always make the same mistake :-)